tracee-ebpf/btfinfo.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"bufio"
+	"errors"
+	"os"
+	"strings"
+	"syscall"
+)
+
+// Helpers
+
+func FileExists(path string) bool {
+	_, err := os.Stat(path)
+	return err == nil
+}
+
+func FileGrepUnique(path string, needle string) string {
+	file, _ := os.Open(path)
+	defer file.Close()
+	scanner := bufio.NewScanner(file)
+	for i := 1; scanner.Scan(); i++ {
+		if strings.Contains(scanner.Text(), needle) {
+			return scanner.Text()
+		}
+	}
+
+	return ""
+}
+
+func ByteToString(all [65]int8) string {
+	var temp [65]byte
+	for i, b := range all {
+		temp[i] = byte(b)
+	}
+	conv := string(temp[:])
+	conv = conv[:strings.Index(conv, "\x00")]
+
+	return conv
+}
+
+func KernelVersion() string {
```

Comment:
shouldn't we return error as well?

Reply:
Hum, only error here would come from:

	err := syscall.Uname(&utsname)

and I had panic() before. Only error for uname(2) is EFAULT if buffer is not valid.. but we're for sure allocating it from the function stack.. so I think we are good.. I'll let you resolve this if you agree, if not I can change.

Reply:
I'll mark this as resolved because Yaniv mentioned existing function: func UnameRelease() which I'll be using.